web/src/components/Collapsible.js

```diff
@@ -28,7 +28,7 @@ class Collapsible extends Component {
       onChange
     } = this.props;
     const isOpen = onChange ? this.props.open : this.state.open;
-    const contentId = kebabCase(title);
+    const contentId = `collapsible-${kebabCase(title)}`;
```

Comment:
Make these IDs a bit more unique, otherwise we could easily accidentally create collisions.  (It actually happened with `executive-summary`!)